pystan/plots.py

```diff
@@ -16,4 +16,4 @@ def traceplot(fit, pars, **kwargs):
     except ImportError:
         logger.critical("matplotlib required for plotting.")
         raise
-    return plots.traceplot(fit.extract(), pars, **kwargs)
+    return plots.traceplot(fit.extract(dtypes=dtypes), pars, **kwargs)
```

Comment:
I decided not to handle dtypes in pystan/plot.py.
This is because this variable is finally passed to stanfit4model's extract method (and initialized to empty dict).
I think it would be better to reduce redunduncy.